src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Mappers/DraftApprenticeshipTests/WhenIMapViewDraftApprenticeshipDetailsToViewModel.cs

```diff
@@ -19,27 +23,29 @@ public class WhenIMapViewDraftApprenticeshipDetailsToViewModel
     {
         private ViewDraftApprenticeshipViewModelMapper _mapper;
         private DraftApprenticeshipRequest _source;
+        public Mock<IOuterApiClient> OuterApiClient;
         private Func<Task<ViewDraftApprenticeshipViewModel>> _act;
-        private GetDraftApprenticeshipResponse _apiResponse;
+        private Infrastructure.OuterApi.Requests.Cohorts.GetDraftApprenticeshipResponse _apiResponse;
         private GetTrainingProgrammeResponse _apiTrainingProgrammeResponse;
 
         [SetUp]
         public void Arrange()
         {
             var fixture = new Fixture();
 
-            _apiResponse = fixture.Build<GetDraftApprenticeshipResponse>().Create();
+            _apiResponse = fixture.Build<Infrastructure.OuterApi.Requests.Cohorts.GetDraftApprenticeshipResponse>().Create();
             _apiTrainingProgrammeResponse = fixture.Build<GetTrainingProgrammeResponse>().Create();
 
             var commitmentsApiClient = new Mock<ICommitmentsApiClient>();
-            commitmentsApiClient.Setup(x =>
-                    x.GetDraftApprenticeship(It.IsAny<long>(), It.IsAny<long>(), It.IsAny<CancellationToken>()))
-                .ReturnsAsync(_apiResponse);
 
             commitmentsApiClient.Setup(x => x.GetTrainingProgramme(_apiResponse.CourseCode, It.IsAny<CancellationToken>()))
                 .ReturnsAsync(_apiTrainingProgrammeResponse);
 
-            _mapper = new ViewDraftApprenticeshipViewModelMapper(commitmentsApiClient.Object);
+            OuterApiClient = new Mock<IOuterApiClient>();
+            OuterApiClient.Setup(x => x.Get<Infrastructure.OuterApi.Requests.Cohorts.GetDraftApprenticeshipResponse>(It.IsAny<GetDraftApprenticeshipRequest>()))
+                .ReturnsAsync(_apiResponse);
+
```

Comment:
We have updated this file using the IOuterAPIClient but I think it should be using the IApprovalService. Also the new fields are not supported by any tests